core/python2Action/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for Python 2 actions, mirrors the Python 3-based pythonAction
+
+FROM python:2.7.13-alpine
+
+# Upgrade and install basic Python dependencies
+RUN apk add --no-cache bash \
+ && apk add --no-cache --virtual .build-deps \
+        bzip2-dev \
+        gcc \
+        libc-dev \
+  && pip install --no-cache-dir gevent==1.2.1 flask==0.12 \
+  && apk del .build-deps
+
+RUN apk add --no-cache \
+        bzip2-dev \
+        gcc \
+        libc-dev \
+        libxslt-dev \
+        libxml2-dev \
+        libffi-dev \
+        openssl-dev
```

Comment:
core/python2Action is meant to [temporary](https://github.com/openwhisk/openwhisk/issues/1184#issuecomment-245334998) replacement core/actionProxy (aka dockerskeleton) and core/pythonAction as we transition them from from Python 2.7 to Python 3.6.  We  can optimize the calls to `apk add` in new core/python2Proxy as you suggest but that will increase the cognitive load and chance of errors for those who need to keep core/python2Proxy in sync with changes made to core/actionProxy and core/pythonProxy.